.github/workflows/build-and-release.yaml

```diff
@@ -0,0 +1,103 @@
+name: Docker build and Release
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+
```

Comment:
You don't need to run the pipeline if PR is closed.

Reply:
Fixed